web_sh/update.sh

```
@@ -0,0 +1,45 @@
+cd $HOME
+backup_dir="$HOME/www/python/src_backup_$(date +%Y%m%d_%H%M%S)"
+
+# Backup existing source if it exists
+if [ -d "$HOME/www/python/src" ]; then
+    mv "$HOME/www/python/src" "$backup_dir" || exit 1
+else
+    echo "No existing source found in $HOME/www/python"
+    exit 1
+fi
+
+TARGET_DIR="$HOME/www/python/src/make2"
```

Shellcheck (0.10.0)

[warning] TARGET_DIR appears unused. Verify use (or export if used externally).

(SC2034)